hxrsnd/macromotor.py

```diff
@@ -170,8 +175,12 @@ def set(self, position, wait=True, verify_move=True, ret_status=True,
             return
 
         # Send the move commands to all the motors
-        self._status = flatten(self._move_towers_and_diagnostics(
+        status = flatten(self._move_towers_and_diagnostics(
```

Comment:
Something I realized after looking at this again is that after we perform a calibration, in theory, we would need to move the diagnostic motor to compensate for the slight position change. However, it may end up that the changes are small enough that it's unnecessary. I tried looking through the code and there isn't a nice way to do it with the current setup, so that's something I should probably rework.